README.md

```diff
@@ -171,6 +190,21 @@ func application(application: UIApplication, openURL url: NSURL, sourceApplicati
 }
 ```
 
+If you want to use your test app during development, in application:didFinishLaunchingWithOptions: you can get the Branch object like this:
+
+```swift
+let branch: Branch = Branch.getTestInstance()
+```
+
+Or
+
+```swift
+let branch: Branch = Branch.getInstance("your test branch key");  // replace with your actual branch key
+```
+
+Either way, we recommend you put a //TODO: or //FIXME landmark to remind you to change back to live app during deployment later.
```

Comment:
Ah, I missed that these were different langs. It would be cool to find some consistency between the two, though. I'm still not real familiar with Swift, but it's unfortunate that those differ.